source/_integrations/sensor.command_line.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Command line Sensor"
+title: "Command Line Sensor"
```

Comment:
The title is synced from the code base, changing it here will be reverted automatically.